src/containers/assignees/reducer.js

```diff
@@ -0,0 +1,61 @@
+import {
+  ADD_ASSIGNEE_TO_CARD, ADD_RESPONSIBLE_TO_CARD, GET_ALL_ASSIGNEES_IN_CARD, GET_RESPONSIBLE_FOR_CARD,
+  REMOVE_ASSIGNEE_FROM_CARD, REMOVE_RESPONSIBLE_FROM_CARD } from './constants'
+
+const initialState = {
+  assignees: [],
+  responsible: null,
+}
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case GET_ALL_ASSIGNEES_IN_CARD:
```

Comment:
You don't need this if you just return the state (an unknown case will fall in the default case). Do the same thing for other useless cases in this file.